CapabilityStatement/CapabilityStatement-England.xml

```diff
@@ -1,7 +1,7 @@
 <CapabilityStatement xmlns="http://hl7.org/fhir">
   <id value="apim-conformance" />
   <url value="https://fhir.hl7.org.uk/CapabilityStatement/England-conformance" />
-  <version value="1.0.0" />
+  <version value="1.1.0" />

```

Comment:
Does this need to update as theres no change to its content?

Reply:
Based on DA decision, will revert the version value for files not actually updated.